go.mod

```diff
@@ -12,8 +12,8 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/gorilla/mux v1.8.0
 	github.com/hashicorp/go-cleanhttp v0.5.1
-	github.com/hashicorp/go-getter v1.5.1
-	github.com/hashicorp/go-retryablehttp v0.6.6
+	github.com/hashicorp/go-getter v1.5.2
```

Comment:
@patilpankaj212 can you omit this go.mod change ? This caused build failure on ARM and willie's PR undid this. 
https://github.com/accurics/terrascan/pull/507/files